### DISCUSSION TITLE
"corresponding plists" for mobileconfigs

### DISCUSSION THREAD

**Discussion by noambernstein** [usnistgov/macos_security#69] [Q&A]
What are the "corresponding plists" in the following quote from the documentation used for?  mobileconfig files can be opened and loaded into system preferences, but it's not obvious what, if anything, is supposed to be done with the associated plist files

"Adding the -p flag to the generate-guidance.py script will generate configuration profiles and the corresponding plists for the rules that have the mobileconfig set to true in the rules YAML file."

**Comment by golbiga** [comment]
This was a feature request #38. When you pass the `-p` flag the plists for the payload under mobileconfig_info will also be generated in `build/{baseline}/mobileconfigs/preferences`. The plist files can then be used in the MDM of your choice to create custom configuration profiles. I'll see if I could word this better in the wiki.

**Reply by noambernstein** [reply]
Thanks.
